docs/pages/deploy-a-cluster/helm-deployments/kubernetes-cluster.mdx

```diff
@@ -165,7 +166,14 @@ to create a public IP for Teleport.
   </TabItem>
 </Tabs>
 
+{/* Disabling the DNS partial until it's updated to cover this scope.
 (!docs/pages/includes/dns.mdx!)
+*/}
+
+Configure DNS entries for `tele.example.com` for traffic to the Teleport Proxy
+web UI, and `*.tele.example.com` for web apps using Application Access. The record
+type and configuration will depend on your Kubernetes provider, see their
```

Comment:
Should we be more explicit here and tell the reader to create CNAME records if `$SERVICE_IP` is a hostname and a A records if it's an IP address?

Reply:
Amended.